src/shared/permissions.ts

```diff
@@ -62,29 +65,26 @@ async function handleInsufficientPrivileges(
   sender: string,
   repo: Context["payload"]["repository"],
   action: string,
-  labelName: string,
-  eventName: string
+  labelName: string
 ) {
-  const { logger, payload } = context;
+  const { logger, config } = context;
   logger.info("Checking access for labels", { repo: repo.full_name, user: sender, labelType });
-  // check permission
-  const { access, user } = context.adapters.supabase;
-  const userId = await user.getUserId(context, sender);
-  const accessible = await access.getAccess(userId, repo.id);
-  if (accessible && accessible.labels?.includes(labelType)) {
-    return true;
-  }
-
-  if (action === "labeled") {
-    await removeLabelFromIssue(context, labelName);
-  } else if (action === "unlabeled") {
-    await addLabelToIssue(context, labelName);
-  }
 
-  if ("issue" in payload && payload.issue) {
-    await addCommentToIssue(context, `@${sender}, You are not allowed to ${eventName} ${labelName}`, payload.issue.number);
-    logger.info("No access to edit label", { sender, label: labelName });
+  if (config.publicAccessControl.protectLabels.some((protectedLabel) => protectedLabel.toLowerCase() === labelType.toLowerCase())) {
+    await postComment(
+      context,
+      logger.error(
+        `@${sender}, you do not have permissions to adjust ${config.publicAccessControl.protectLabels.map((label) => `\`${label}\``).join(", ")} labels.`,
```

Comment:
I think it's redundant to write the users name in any of the warning and error messages.

Reply:
Sure, can remove the name.

Reply:
https://github.com/ubiquity-os-marketplace/daemon-pricing/pull/66/commits/8ec54ff80b2c92ee549cbd7f0935a3fc0d8adeea